core/state/pruner/pruner.go

```diff
@@ -464,6 +534,17 @@ func dumpRawTrieDescendants(db ethdb.Database, root common.Hash, output *stateBl
 	return nil
 }
 
+func keybytesToHex(str []byte) []byte {
```

Comment:
instead of copying the keybytesToHex can we export `trie.keyBytesToHex` and use it here? 

e.g. by just adding in `trie/encoding.go`:
```
func KeybytesToHex(str []byte) []byte {
	return keybytesToHex(str)
}
```